diff --git a/node_modules/@types/readable-stream/index.d.ts b/node_modules/@types/readable-stream/index.d.ts
index 0ef4716..8337253 100644
--- a/node_modules/@types/readable-stream/index.d.ts
+++ b/node_modules/@types/readable-stream/index.d.ts
@@ -8,6 +8,7 @@
 /// <reference types="node" />
 
 import * as SafeBuffer from "safe-buffer";
+import type * as NodeStream from "stream";
 
 declare class StringDecoder {
     constructor(encoding?: BufferEncoding | string);
@@ -658,6 +659,9 @@ declare namespace _Readable {
             options?: { signal: AbortSignal },
         ): T;
     }
+
+    const finished: typeof NodeStream.finished;
+    const pipeline: typeof NodeStream.pipeline;
 }
 
 export = _Readable;
